<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HexLoaderClass</class>
 <widget class="QMainWindow" name="HexLoaderClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>840</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>HexLoader</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTableWidget" name="tableWidget_hex">
      <property name="editTriggers">
       <set>QAbstractItemView::AnyKeyPressed</set>
      </property>
      <property name="gridStyle">
       <enum>Qt::DashLine</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>840</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_open"/>
    <addaction name="action_close"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Other</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_openfile"/>
   <addaction name="separator"/>
   <addaction name="action_save"/>
   <addaction name="action_saveas"/>
   <addaction name="separator"/>
   <addaction name="action_undo"/>
   <addaction name="action_redo"/>
   <addaction name="separator"/>
   <addaction name="action_find"/>
  </widget>
  <action name="action_open">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="action_close">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="action_undo">
   <property name="icon">
    <iconset resource="hexloader.qrc">
     <normaloff>:/HexLoader/Resources/revoke.png</normaloff>:/HexLoader/Resources/revoke.png</iconset>
   </property>
   <property name="text">
    <string>undo</string>
   </property>
   <property name="toolTip">
    <string>undo(ctrl+z)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_redo">
   <property name="icon">
    <iconset resource="hexloader.qrc">
     <normaloff>:/HexLoader/Resources/revert.png</normaloff>:/HexLoader/Resources/revert.png</iconset>
   </property>
   <property name="text">
    <string>redo</string>
   </property>
   <property name="toolTip">
    <string>redo(ctrl+y)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="action_save">
   <property name="icon">
    <iconset resource="hexloader.qrc">
     <normaloff>:/HexLoader/Resources/save.png</normaloff>:/HexLoader/Resources/save.png</iconset>
   </property>
   <property name="text">
    <string>save</string>
   </property>
   <property name="toolTip">
    <string>save(ctrl+s)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_saveas">
   <property name="icon">
    <iconset resource="hexloader.qrc">
     <normaloff>:/HexLoader/Resources/saveas.png</normaloff>:/HexLoader/Resources/saveas.png</iconset>
   </property>
   <property name="text">
    <string>saveas</string>
   </property>
   <property name="toolTip">
    <string>save as(ctrl+a)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action_find">
   <property name="icon">
    <iconset resource="hexloader.qrc">
     <normaloff>:/HexLoader/Resources/search.png</normaloff>:/HexLoader/Resources/search.png</iconset>
   </property>
   <property name="text">
    <string>find</string>
   </property>
   <property name="toolTip">
    <string>find(ctrl+f)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="action_openfile">
   <property name="icon">
    <iconset resource="hexloader.qrc">
     <normaloff>:/HexLoader/Resources/open_file.png</normaloff>:/HexLoader/Resources/open_file.png</iconset>
   </property>
   <property name="text">
    <string>openfile</string>
   </property>
   <property name="toolTip">
    <string>open file(ctrl+o)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="hexloader.qrc"/>
 </resources>
 <connections/>
</ui>
